cmd/draupnir-finalise-image

```diff
@@ -94,8 +94,26 @@ wal_level = 'minimal'
 fsync = 'off'
 EOF
 
+LOG_FILE="/var/log/postgresql/image_${ID}"
+
 # Start postgres
-sudo -u postgres $PG_CTL -w -D "$UPLOAD_PATH" -o "-p $PORT" -l "/var/log/postgresql/image_$ID" start
+sudo -u postgres $PG_CTL -w -D "$UPLOAD_PATH" -o "-p $PORT" -l "${LOG_FILE}" start
+
```

Comment:
I would love to make these go scripts tbh. I know it becomes more verbose but on balance I find you write those scripts once and then they very rarely go wrong, while maintaining proper logging and all the good stuff. I hope we get the time to revisit this project one day!